packages/eslint-config-react-ts/package.json

```diff
@@ -23,6 +23,7 @@
 	},
 	"devDependencies": {
 		"eslint": ">=8.14",
+		"eslint-plugin-typescript-enum": "^2.1.0",
```

Comment:
Should this be added to peerDeps also?
npm v7 will install peer deps by default, otherwise we need to install it manually by adding it to this command https://github.com/infinum/js-linters/tree/master/packages/eslint-config-react-ts#usage

```
npm install --save-dev eslint eslint-plugin-jsx-a11y eslint-plugin-react eslint-plugin-react-hooks @typescript-eslint/eslint-plugin @typescript-eslint/parser @infinumjs/eslint-config-react-ts
```

Reply:
More details on this problem https://eslint.org/blog/2022/08/new-config-system-part-1/#shareable-configs-and-dependencies

Reply:
@gbareza ping!
Could you give me your opinion on this info, maybe I can finish this PR if you don't have time.

Reply:
Sry, didn't have time to check this. If you think we should do this, go ahead. Otherwise, I'll try to find some time in next few weeks to check it out.